keanu-python/keanu/vertex/vertex_helpers.py

```diff
@@ -0,0 +1,54 @@
+from typing import Callable, Dict, Any, Collection
```

Comment:
the name `vertex_helpers` is very generic - I'd rather call this `vertex_casting` to avoid it becoming a catch-all for miscellaneous static functions.

Reply:
I sort of wanted it to be a catch all as it happens (I find bigger modules have less tendency to cause issues with the dependency graph over time :) but happy to rename.